### ISSUE TITLE
Adding an Helm repo from additionalConfigMaps

### ISSUE THREAD

**Issue by johnniee** [Oteemo/charts#301]
Hi,
I'm using `additionalConfigMaps` to add my MavenGroup repos. Now i have the need to add a helm repo as well.
I couldn't find the syntax to do that anywhere. Can you please assist?

Thanks!

**Comment by abdennour** [comment]
Yeah! the documentation of this value (additionalConfigMaps) is very weak or basically, it's absent , except that comment about an example of MavenProxy
